src/pages/ScholarshipForm.jsx

```diff
@@ -25,27 +29,52 @@ function ScholarshipForm() {
       description: '',
       amount: {
         type: null,
-        minAmount: 0,
-        maxAmount: 0,
+        min: 0,
+        max: 0,
       },
       website: '',
     },
     validationSchema,
     onSubmit: (values, { setSubmitting, resetForm }) => {
       setSubmitting(true);
-      alert(JSON.stringify(values, null, 2));
-      setSubmitting(false);
-      resetForm();
+      Scholarships.new(values)
+        .save()
+        .then(() => {
```

Comment:
The result here btw is a Scholarship object so you could do something with that. E.g. 

```jsx
        .then((scholarship) => {
          const link = /* generate from scholarship.id like the scholarships page does for its results */;
          setSubmissionAlert(<SnackBar ...> /* Somewhere use the link/title/etc. */</SnackBar>);
        });
```

Reply:
regarding this, the `scholarship` parameter only returns the object that was submitted.

Reply:
Yes, where `scholarship.data` should match the field values and `scholarship.id` should be the ID of the scholarhip.